muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/rest/WorkspaceRESTService.java

```diff
@@ -324,16 +324,23 @@ public Response listWorkspaces(
       }
       UserEntity loggedUserEntity = sessionController.getLoggedUserEntity();
       workspaceEntities = workspaceVisitController.listEnrolledWorkspaceEntitiesByMinVisitsOrderByLastVisit(loggedUserEntity, minVisits);
-    } else {
+    }
+    else {
       if (userIdentifier != null) {
         if (includeArchivedWorkspaceUsers) {
           workspaceEntities = workspaceUserEntityController.listWorkspaceEntitiesByUserIdentifierIncludeArchived(userIdentifier);
         } else {
           workspaceEntities = workspaceUserEntityController.listWorkspaceEntitiesByUserIdentifier(userIdentifier);
         }
-      } else if (userEntity != null) {
+      }
+      else if (userEntity != null) {
         workspaceEntities = workspaceUserEntityController.listWorkspaceEntitiesByUserEntity(userEntity);
       }
```

Comment:
Should we have a separate permission for listing all workspaces including unpublished ones.